### ISSUE TITLE
message in Exception not escaped

### ISSUE THREAD

**Issue by johanvos** [jtulach/bck2brwsr#22]
The following code snippet compiles, but will give an error at runtime:
```
 throw new NegativeArraySizeException("Can't resize");
```
It will generate:
```
var stA0 = new vm.java_lang_NegativeArraySizeException;
var stA1 = stA0;var stA2 = 'Can't resize to negative value: '
```

( the `'` is not escaped)

**Comment by jtulach** [comment]
Simple test seems to work. I am not sure how to reproduce the problem:
```
bck2brwsr$ git diff
diff --git a/rt/emul/compacttest/src/test/java/org/apidesign/bck2brwsr/tck/ExceptionsTest.java b/rt/emul/compacttest/src/test/java/org/apidesign/bck2brwsr/tck/ExceptionsTest.java
index 47b11d64..c9fc09b8 100644
--- a/rt/emul/compacttest/src/test/java/org/apidesign/bck2brwsr/tck/ExceptionsTest.java
+++ b/rt/emul/compacttest/src/test/java/org/apidesign/bck2brwsr/tck/ExceptionsTest.java
@@ -68,10 +68,24 @@ public class ExceptionsTest {
         }
     }
     
+    @Compare
+    public String escapeMessageText() throws Exception {
+        try {
+            return MyException.negativeArray();
+        } catch (RuntimeException ex) {
+            return ex.getMessage();
+        }
+    }
+
+
     static class MyException extends Exception {
         public MyException(String message) {
             super(message);
         }
+
+        static String negativeArray() throws NegativeArraySizeException {
+            throw new NegativeArraySizeException("Can't resize");
+        }
     }
```

**Comment by jtulach** [comment]
Fixed by 2453c61
